### PR TITLE
Add ID Token support to Web Embed

### PR DESCRIPTION
**Pull request by atikenny**
Implements: [APP-90](https://formsort.atlassian.net/browse/APP-90)

Adds the ability to provide an ID Token for authenticated flows.

### CODE REVIEW

File: packages/constants/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@formsort/constants",
-  "version": "1.4.0",
+  "version": "1.5.0",
```
Comment (lizraeli) [inline packages/constants/package.json:3]: The packages are upgraded automatically when running `lerna publish`

File: packages/web-embed-api/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@formsort/web-embed-api",
-  "version": "2.0.3",
+  "version": "2.1.0",
```
Comment (lizraeli) [inline packages/web-embed-api/package.json:3]: Same comment as above for package versions

**Comment by lizraeli** [comment]
Looks good. See comment about package upgrades. Could you link to the PR where we add the embed token request/response to the flow?

**Review by lizraeli — APPROVED** [review]
🚀

**Comment by atikenny** [comment]
> Looks good. See comment about package upgrades. Could you link to the flow PR/code where we handle the embed token request?

It's an ongoing development, will link to it once it is fianlized.

**Pull request merged**